swap_meet/vendor.py

```diff
@@ -1,2 +1,117 @@
+from .item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    # Define attributes for Vendor instances
+    def __init__(self, inventory = None):
+        # If no value provided for inventory, attribute inventory takes on an empty list 
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+        
+    def add(self, item_add):
+        # Add item provided to attribute inventory
+        self.inventory.append(item_add)
+        # Return this added item
+        return item_add
+    
+    def remove(self, item_remove):
+        # If item to be removed doesn't exist in attribute inventory, return False.
+        # Otherwise, remove this item from inventory and return this removed item
+        if item_remove not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item_remove)
+            return item_remove
+        
+    def get_by_id(self, id):
+        # Create an Item instance passing in an id value
+        test_item = Item(id)
+        # Loop through list of instances stored in attribute inventory
+        for item in self.inventory:
+            # If attribute id in an instance has same value as attribute id in Item instance
+            # Return this instance. Otherwise, return None
+            if item.id == test_item.id:
```

Comment:
I'd recommend not creating the fake `Item` with `id`. You could just make this:

```suggestion
            if item.id == id:
```